app/lib/dynamicRequire.js

```diff
@@ -125,5 +125,7 @@ module.exports = {
   'views/courses/RestrictedToStudentsView': function () { return import(/* webpackChunkName: "RestrictedToStudentsView" */ 'views/courses/RestrictedToStudentsView') },
   'views/user/CertificatesView': function () { return import(/* webpackChunkName: "CertificatesView" */ 'views/user/CertificatesView') },
   'views/user/AnonCertificatesView': function () { return import(/* webpackChunkName: "CertificatesView" */ 'views/user/AnonCertificatesView') },
-  'views/LicensorView': function() { return import(/* webpackChunkName: "LicensorView" */ 'views/LicensorView')}
+  'views/LicensorView': function() { return import(/* webpackChunkName: "LicensorView" */ 'views/LicensorView')},
+
+  'views/core/SingletonAppVueComponentView': function () { return import(/* webpackChunkName: "core" */ 'views/core/SingletonAppVueComponentView').then(m => m.default) },
```

Comment:
Are we going to use this for every page we route using vue-router? How will the webpack chunking work in this case?

Reply:
Discussed offline:

- Name chunk `vue` for now - plan is to put all Vue components in a single chunk for now
- Later: use native async Vue components / Vue router to load specific chunks of Vue components
Add comment describinb this

Reply:
Added comment